test/multiverse/suites/rake/rake_test.rb

```diff
@@ -59,6 +59,9 @@ def test_records_transaction_metrics
       end
 
       def test_records_transaction_trace
+        # TODO: JRuby 9.4.0.0 fails this test only via GHA
```

Comment:
Will this TODO be resolved as part of this ticket? If not, could you make an issue for it?

Reply:
New TODO tracked with https://github.com/newrelic/newrelic-ruby-agent/issues/1664

Reply:
Thank you for turning the TODOs into issues!